src/runner/browser-env/vite/browser-modules/expect.ts

```diff
@@ -0,0 +1,96 @@
+import { expect, type Expect } from "expect";
+import { ASYMMETRIC_MATCHER } from "./constants.js";
+import { BrowserEventNames, type BrowserRunExpectMatcherPayload, MockMatcherFn } from "./types.js";
+
+export const initExpect = (): Expect => {
+    const mockedMatchers = mockMatchers(window.__testplane__.expectMatchers);
```

Comment:
Inside `expectMatchers` stored the matchers from `expect-webdriverio` which must be run in nodejs runtime